src/main/java/com/codeless/api/automation/entity/security/View.java

```diff
@@ -0,0 +1,6 @@
+package com.codeless.api.automation.entity.security;
+
+public class View {
```

Comment:
if it is used in DTO, please move it to dto package in backend.

Reply:
moved to backend